src/common-imports.ts

```diff
@@ -0,0 +1,32 @@
+import Vue from 'vue'
+
+import * as commonComponents from '@/components/common'
+import * as interactionDesignerComponents from '@/components/interaction-designer'
+import * as blockEditorsComponents from '@/components/interaction-designer/block-editors'
+import * as blockTypesComponents from '@/components/interaction-designer/block-types'
+import * as flowEditorsComponents from '@/components/interaction-designer/flow-editors'
+import * as flowImportComponents from '@/components/interaction-designer/flow-editors/import'
+import * as toolbarComponents from '@/components/interaction-designer/toolbar'
+import * as blocksComponents from '@/components/interaction-designer/blocks'
+import * as blockResourceEditorsComponents from '@/components/interaction-designer/resource-editors'
+import * as resourceEditorComponents from '@/components/resource-editor'
+
+require('./font-awesome-icon')
+
+export default function registerCustomComponents(extra = {}) {
+  Object.entries({
+    ...commonComponents,
+    ...interactionDesignerComponents,
+    ...blockEditorsComponents,
+    ...blockTypesComponents,
+    ...flowEditorsComponents,
+    ...flowImportComponents,
+    ...toolbarComponents,
+    ...blocksComponents,
+    ...blockResourceEditorsComponents,
+    ...resourceEditorComponents,
+    ...extra,
+  }).forEach(([name, Component]) => {
+    Vue.component(name, Component)
```

Comment:
This is great. Thanks for making this change. Only issue is that this isn't building for my currently?

```
ERROR in /home/jacob/voto/flow-builder/src/common-imports.ts(30,25):
30:25 No overload matches this call.
  The last overload gave the following error.
    Argument of type '((builder: Vue) => true | void) | typeof ValidationMessage | typeof BlockOutputBranchingConfig | typeof OutputBranchingType | ... 72 more ... | { ...; }' is not assignable to parameter of type 'ComponentOptions<Vue, DefaultData<Vue>, DefaultMethods<Vue>, DefaultComputed, PropsDefinition<DefaultProps>, DefaultProps> | undefined'.
      Type '(builder: Vue) => true | void' has no properties in common with type 'ComponentOptions<Vue, DefaultData<Vue>, DefaultMethods<Vue>, DefaultComputed, PropsDefinition<DefaultProps>, DefaultProps>'.
    28 |     ...extra,
    29 |   }).forEach(([name, Component]) => {
  > 30 |     Vue.component(name, Component)
       |                         ^
    31 |   })
    32 | }
```